src/main/java/ch/uzh/ifi/hase/soprafs22/game/player/Player.java

```diff
@@ -71,9 +71,10 @@ public RegisteredUser getRegisteredUser() {
     }
 
     @Override
-    public void linkRegisteredUser(RegisteredUser registeredUser){
+    public void linkRegisteredUser(RegisteredUser registeredUser, String token){
```

Comment:
if we link a registered user, we probably should use the token of the registered user

Reply:
Thanks!, done